index.html

```diff
@@ -6,10 +6,125 @@
+                </a>
+            </div>
+            <div class="main-menu">
+                <ul id="menu">
+                    <li><a href="about.html" class="active menu-link">About</a></li>
+                    <li><a href="#program" class="menu-link">Program</a></li>
+                    <li><a href="#register" class="menu-link">Register</a></li>
+                    <li><a href="#partner" class="menu-link">Sponsor</a></li>
+                    <li><a href="#" class="menu-link">News</a></li>
+                </ul>
+                <button class="menu-button">Register for VarCamp</button>
+            </div>
+        </nav>
+    </header>
+    <section id="headline">
+        <img id="hambuger" src="./images/icons/list.svg" alt="Hambuger Icon">
+        <p class="welcome">"Hello! Sharing World"</p>
+        <h1 class="headline-title">
+            Varcamp in Myanmar 2023
+        </h1>
+        <div class="about-description">
+            VarCamp (virtual barcamp) is a type of conference or workshop that is organized and hosted online, rather
+            than in
+            person. The format of a virtual barcamp is similar to that of an in-person barcamp, with participants
+            presenting talks,
+            leading discussions, and participating in collaborative activities.
+        </div>
+        <h2 class="headline-date">2023.03.18(SAT) ~ 19(SUN)</h2>
+        <h3 class="headline-place">@Online Virtual Conference</h3>
+    </section>
+    <section id="program">
+        <h2 class="program-main-title">Main Program</h2>
+        <hr class="line">
+        <div class="program-main-card">
+            <div class="program-card">
+                <img src="./images/icons/lecture.svg" alt="Lecture svg" class="program-image">
+                <h4 class="program-title">Lecture</h4>
+                <p>
+                    Listen to the speakers from various companies about the messages of sharing and opening.
+                </p>
+            </div>
+            <div class="program-card">
+                <img src="./images/icons/forum.svg" alt="forum svg" class="program-image">
+                <h4 class="program-title">Forum</h4>
+                <p>
+                    Have the time to share your thoughts and opinions with experts for each topic.
+                </p>
+            </div>
+            <div class="program-card">
+                <img src="./images/icons/workshop.svg" alt="Workshop svg" class="program-image">
+                <h4 class="program-title">Workshop</h4>
+                <p>
+                    Try creating your own work using open source license rather than just watching.
+                </p>
+            </div>
+            <div class="program-card">
+                <img src="./images/icons/communication.svg" alt="Communication svg" class="program-image">
+                <h4 class="program-title">Discussion</h4>
```

Comment:
- [optional] please, some of the text in the main program section are overflowing the card. Please fix that. 
![Screenshot (881)_LI](https://user-images.githubusercontent.com/65298048/225638353-21a3f8a2-468a-4e0d-92b4-7d7bf643b314.jpg)

Reply:
Fixed.